runner/file.go

```diff
@@ -460,6 +461,41 @@ func transformAvroFile(in string, out *ResultWriter) error {
 	return transformAvro(r, out)
 }
 
+func transformYAML(in *bufio.Reader, out *ResultWriter) error {
+	jw := out.w.(*JSONResultItemWriter)
+	jw.raw = true
+	o := jw.bfd
+	enc := jsonNewEncoder(o)
+	dec := yaml.NewDecoder(in)
```

Comment:
There's no version of YAML that is row oriented AFAIK. So the safer thing to do here is drop the loop entirely and just:

```
enc := jsonNewEncoder(o)
dec := yaml.NewDecoder(in)
var a any
err := dec.Decode(&a)
if err != nil {
	return err
}

err = enc.Encode(a)
return err
```

I think

Reply:
Fixed!